cli/src/command/host.rs

```diff
@@ -116,19 +116,6 @@ fn setup_crate(host_path: PathBuf, rev: Option<String>, tag: Option<String>) ->
         ],
     )?;
 
-    // add postcard because it is used for (de)serializing from/to the input/output tapes
```

Comment:
We want to leave this in for now to not break existing programs. We can remove it eventually.